contracts/contracts/layer_0/adjustBalances/AdjustBalancesStorageWrapperRead.sol

```diff
@@ -388,6 +384,98 @@ contract AdjustBalancesStorageWrapperRead is
         factor_ = _abaf / _labaf;
     }
 
+    function _getABAF() internal view virtual returns (uint256) {
```

Comment:
The previous name is preferred. Although ABAF is an acronym, camelCase is more suitable due to its familiarity.